tensorflow/lite/g3doc/inference_with_metadata/task_library/overview.md

```diff
@@ -173,6 +173,33 @@ std::vector<QaAnswer> results = answerer->Answer(context_of_question, question_t
 Explore more advanced accelerator settings
 [here](https://github.com/tensorflow/tensorflow/blob/1a8e885b864c818198a5b2c0cbbeca5a1e833bc8/tensorflow/lite/experimental/acceleration/configuration/configuration.proto).
 
+### Example usage of Coral Edge TPU in Python
+
+Configure Coral Edge TPU in the base options of the task. For example, you can set
+up Coral Edge TPU in `ImageClassifier` as follows:
+
+```python
+# Imports
+from tflite_support.task import vision
+from tflite_support.task import core
+from tflite_support.task import processor
+
+# Initialize options and turn on Coral Edge TPU delegation.
+base_options = core.BaseOptions(
```

Comment:
To make the doc simple, please keep only the required parameters (file_name and use_coral). ClassificationOptions is also optional so please remove it.